p2p/src/net/libp2p/constants.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://spdx.org/licenses/MIT
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author(s): A. Altonen
+
+//! Libp2p constants
+//!
+//! See protocol specification for more details
+
+use std::time::Duration;
+
+/// Gossipsub configuration
+// TODO: config or spec?
+pub const GOSSIPSUB_HEARTBEAT: Duration = Duration::from_secs(10);
+pub const GOSSIPSUB_MAX_TRANSMIT_SIZE: usize = 2 * 1024 * 1024;
```

Comment:
I think this should be determined in a more consistent way to be compatible with block size. I don't know how yet though.

Reply:
We could have a global constants where this and the block size could both be. Making this easily configurable is IMO asking for trouble because you'll be banned immediately if you send too a message that's too large. Can this be resolved and/or should we move this discussion to a Github issue?

Reply:
Yes, please. Create an issue for this.

Reply:
https://github.com/mintlayer/mintlayer-core/issues/250